app/src/main/res/values/strings.xml

```diff
@@ -559,10 +559,10 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_religion_for_wayside_shrine_title">What religion is represented at this shrine?</string>
     <string name="quest_parking_fee_title">Do you have to pay to park here?</string>
     <string name="quest_surface_value_unhewn_cobblestone">Unhewn cobblestone</string>
-    <string name="quest_parking_access_title">Is it restricted who may park here?</string>
-    <string name="quest_parking_access_private">It is private</string>
-    <string name="quest_parking_access_yes">It is open to the public (with or without fee)</string>
-    <string name="quest_parking_access_customers">It is only for customers</string>
+    <string name="quest_parking_access_title2">Who may park here? Parking may be free or paid.</string>
+    <string name="quest_access_yes">Open to general public</string>
+    <string name="quest_access_customers">Open only to people visiting a specific place (e.g. customers)</string>
+    <string name="quest_access_private">Not open to general public</string>
```

Comment:
- Changed all the answers to remove the verbs ("Open to"), to match the updated question phrasing.
    - This gives us some extra characters to work with
- Adapted `=private` phrasing from the [wiki `key:access` page](https://wiki.openstreetmap.org/wiki/Key:access#List_of_possible_values) ("Only with individual permission"), which I like.
- Use the word *Any* to describe `=customers`; move "only" into the parentheses, so it's still present in the answer. That's the language we've been using to describe the tag here ("Anyone, but only if they're using the associated facility"); no need to complicate things.

#### Who is allowed to park here? Parking may be free or paid.

```suggestion
    <string name="quest_access_yes">Any member of the general public</string>
    <string name="quest_access_customers">Any person visiting a specific place (e.g. customers only)</string>
    <string name="quest_access_private">Only individual(s) with permission</string>
```